### ISSUE TITLE
【Bug】Some temp data are lost and makes JIT error(s)

### ISSUE THREAD

**Issue by mcmartin25** [tobychui/cyinput#26]
**Desktop:**
 - OS: [Windows 10 HK Version 1809 OS Build 17763.557] 

This message pops up after my computer wakes up.

![image](https://user-images.githubusercontent.com/45849014/60381659-bab37c00-9a8a-11e9-9999-7f9f7aef5557.png)

It happens after my computer wakes up from sleep mode and type something using cyinput.

For example, the message appears with different file path. If I press 5 on the number pad, the file path will be "C:\Users\Martin\AppData\Local\Temp\cyinput\img\5\1.png".

When I click `結束` it will terminate itself, or else will continue to run. However, it still continue to pop up these error when I press 1-8 on the number pad, as folders named 0 and 9 are intact.

![image](https://user-images.githubusercontent.com/45849014/60381635-7b852b00-9a8a-11e9-9375-c5aa854c7574.png)

I found a temporary solution:
1. Close cyinput
2. Go to `%userprofile%\appdata\local\Temp`
3. Delete `cyinput` folder
4. Restart cyinput

**This works. However, it will pop same error message again after days.**

**Additional context**
Here is the error infomation when I press 6 on the number pad. Hope this helps to debug.
```
如需叫用 Just-In-Time (JIT) 偵錯的詳細資料，
請參閱本訊息結尾處 (而非這個對話方塊) 的資訊。

************** 例外狀況文字 **************
System.IO.FileNotFoundException: C:\Users\Martin\AppData\Local\Temp\cyinput\img\6\1.png
   於 System.Drawing.Image.FromFile(String filename, Boolean useEmbeddedColorManagement)
   於 System.Drawing.Image.FromFile(String filename)
   於 svchost.Form1.paintPictureBoxes(String id)
   於 svchost.Form1.handlePage()
   於 svchost.Form1.appendToCharSet(String number)
   於 svchost.Form1.HandleHotkey(Int32 keycode)
   於 svchost.Form1.WndProc(Message& m)
   於 System.Windows.Forms.Control.ControlNativeWindow.OnMessage(Message& m)
   於 System.Windows.Forms.Control.ControlNativeWindow.WndProc(Message& m)
   於 System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, Int32 msg, IntPtr wparam, IntPtr lparam)


************** 已載入的組件 **************
mscorlib
    組件版本: 4.0.0.0
    Win32 版本: 4.7.3416.0 built by: NET472REL1LAST_B
    程式碼庫: file:///C:/Windows/Microsoft.NET/Framework/v4.0.30319/mscorlib.dll
----------------------------------------
cyinput
    組件版本: 1.0.3.0
    Win32 版本: 1.0.0.0
    程式碼庫: file:///D:/Program%20Files/cyinput.exe
----------------------------------------
Microsoft.VisualBasic
    組件版本: 10.0.0.0
    Win32 版本: 14.7.3190.0 built by: NET472REL1LAST_C
    程式碼庫: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/Microsoft.VisualBasic/v4.0_10.0.0.0__b03f5f7f11d50a3a/Microsoft.VisualBasic.dll
----------------------------------------
System.Windows.Forms
    組件版本: 4.0.0.0
    Win32 版本: 4.7.3324.0 built by: NET472REL1LAST_C
    程式碼庫: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Windows.Forms/v4.0_4.0.0.0__b77a5c561934e089/System.Windows.Forms.dll
----------------------------------------
System
    組件版本: 4.0.0.0
    Win32 版本: 4.7.3416.0 built by: NET472REL1LAST_B
    程式碼庫: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System/v4.0_4.0.0.0__b77a5c561934e089/System.dll
----------------------------------------
System.Drawing
    組件版本: 4.0.0.0
    Win32 版本: 4.7.3190.0 built by: NET472REL1LAST_C
    程式碼庫: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Drawing/v4.0_4.0.0.0__b03f5f7f11d50a3a/System.Drawing.dll
----------------------------------------
System.Configuration
    組件版本: 4.0.0.0
    Win32 版本: 4.7.3324.0 built by: NET472REL1LAST_C
    程式碼庫: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Configuration/v4.0_4.0.0.0__b03f5f7f11d50a3a/System.Configuration.dll
----------------------------------------
System.Core
    組件版本: 4.0.0.0
    Win32 版本: 4.7.3429.0 built by: NET472REL1LAST_C
    程式碼庫: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Core/v4.0_4.0.0.0__b77a5c561934e089/System.Core.dll
----------------------------------------
System.Xml
    組件版本: 4.0.0.0
    Win32 版本: 4.7.3190.0 built by: NET472REL1LAST_C
    程式碼庫: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Xml/v4.0_4.0.0.0__b77a5c561934e089/System.Xml.dll
----------------------------------------
Accessibility
    組件版本: 4.0.0.0
    Win32 版本: 4.7.3190.0 built by: NET472REL1LAST_C
    程式碼庫: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/Accessibility/v4.0_4.0.0.0__b03f5f7f11d50a3a/Accessibility.dll
----------------------------------------
System.Runtime.Remoting
    組件版本: 4.0.0.0
    Win32 版本: 4.7.3190.0 built by: NET472REL1LAST_C
    程式碼庫: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Runtime.Remoting/v4.0_4.0.0.0__b77a5c561934e089/System.Runtime.Remoting.dll
----------------------------------------
VBHotkeyLib
    組件版本: 1.0.0.0
    Win32 版本: 1.0.0.0
    程式碼庫: file:///D:/Program%20Files/cyinput.exe
----------------------------------------
System.IO.Compression.FileSystem
    組件版本: 4.0.0.0
    Win32 版本: 4.7.3190.0
    程式碼庫: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.IO.Compression.FileSystem/v4.0_4.0.0.0__b77a5c561934e089/System.IO.Compression.FileSystem.dll
----------------------------------------
mscorlib.resources
    組件版本: 4.0.0.0
    Win32 版本: 4.7.3190.0 built by: NET472REL1LAST_C
    程式碼庫: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/mscorlib.resources/v4.0_4.0.0.0_zh-Hant_b77a5c561934e089/mscorlib.resources.dll
----------------------------------------
System.Windows.Forms.resources
    組件版本: 4.0.0.0
    Win32 版本: 4.7.3190.0 built by: NET472REL1LAST_C
    程式碼庫: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Windows.Forms.resources/v4.0_4.0.0.0_zh-Hant_b77a5c561934e089/System.Windows.Forms.resources.dll
----------------------------------------

************** JIT 偵錯 **************
若要啟用 Just-In-Time (JIT) 偵錯功能，則必須在
此應用程式或電腦的 .config 檔案中，設定
system.windows.forms 區段內的 jitDebugging 值。
且該應用程式也必須在啟用偵錯的狀態下進行
編譯。

例如:

<configuration>
    <system.windows.forms jitDebugging="true" />
</configuration>

當 JIT 偵錯功能啟用後，會將所有未處理的例外狀況
傳送給電腦上已註冊的 JIT 偵錯工具進行處
理，而不是使用這個對話方塊來處理。
```
Thanks a lot!

**Comment by tobychui** [comment]
Hi, thanks for your feedback.

Just a quick question, do you use any tools or software that help you to "clean up" more space in your computer? cyinput store the images in the TEMP folder you shown in the image above. I am guessing if there are extra software(s) that clean up the TEMP folder. 

Moreover, you mentioned that "it will pop same error message again after days.". May I know have you turned off / restarted your computer within these period or you just leave it as "sleep mode" for a few days? 
Your answers can help us fix the bug as soon as possible. Thanks :)

**Comment by mcmartin25** [comment]
Thanks for quick response.

> Just a quick question, do you use any tools or software that help you to "clean up" more space in your computer?

I do not use any cleanup tool on this computer.

> May I know have you turned off / restarted your computer within these period or you just leave it as "sleep mode" for a few days?

In fact, cyinput works normally after every boot up, I turn my computer to sleep mode usually for about a few minutes to a maximum of an hour or so, then turn it back on, and cyinput may have problems sometimes.

In simple, I have turned off / restarted my computer within these period, and I did not keep it as sleep mode for a few days.
